sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -31,10 +31,10 @@ internal sealed class AzureCoreEventSource : EventSource
 
         public static AzureCoreEventSource Singleton { get; } = new AzureCoreEventSource();
 
-        [Event(RequestEvent, Level = EventLevel.Informational, Message = "Request [{0}] {1} {2}\r\n{3}")]
-        public void Request(string requestId, string method, string uri, string headers)
+        [Event(RequestEvent, Level = EventLevel.Informational, Message = "Request [{0}] {1} {2}\r\n{3}, resource provider namespace: {4}")]
```

Comment:
Now I wonder if we should just include the client assembly instead. Might be less confusing to customers.

Reply:
Updated this to include client assembly name.